addons/http_routing/views/http_routing_template.xml

```diff
@@ -108,21 +108,24 @@
                     <section class="pt128 pb104">
                         <div class="container">
                             <div class="row">
-                                <div class="col-md-6 text-center text-md-right my-md-auto mb-5 mb-md-0">
-                                    <img class="img img-fluid" src="/http_routing/static/src/img/404.svg"/>
+                                <div class="col-lg-4 pb32">
```

Comment:
As well spotted by @JKE-be , can you add those back or fix it ?
Now the layout is broken if you add text
Before
![image](https://user-images.githubusercontent.com/30048408/101173928-7e068f80-3643-11eb-8035-4af433a4c8da.png)
Now
![image](https://user-images.githubusercontent.com/30048408/101173970-8bbc1500-3643-11eb-9c51-baf43acbed72.png)

Reply:
Do you mean adding a `align-self-center` in the column and setting the two columns to `col-lg-6`?

<img width="1667" alt="image" src="https://user-images.githubusercontent.com/43962154/101176559-f15dd080-3646-11eb-913d-e2785a7ef240.png">

What do you think, @Cocographique ?

Reply:
@karlosvalverde just the vertical align yes, not the col-lg. overall your layout is cleaner, it is a bit less 'brouillon'

Reply:
If it's about vertical alignment, you can add `.s_image_text `on the `section` 
and `.align-items-center` on the parent `.row`
[https://github.com/odoo/odoo/blob/4f1c6a1b14e2d969c4ca40b9c0181f93163dda2f/addons/website/views/snippets/snippets.xml#L1044](https://github.com/odoo/odoo/blob/4f1c6a1b14e2d969c4ca40b9c0181f93163dda2f/addons/website/views/snippets/snippets.xml#L1044)

Reply:
Done!